src/main/java/org/fcrepo/spec/testsuite/crud/Container.java

```diff
@@ -729,4 +732,571 @@ public void ldpDirectContainerMayAllowLdpIsMemberRelationToBeUpdatedByPatch() {
+                  "https://fcrepo.github.io/fcrepo-specification/#ldpic",
+                  ps);
+        //throw skip exception if indirect containers not supported
+        skipIfIndirectContainersNotSupported();
+
+        final String hasMemberPredicate = "http://example.org/ldp/member";
+        final String body = "<> <" + LDP_HAS_MEMBER_RELATION_PREDICATE + "> <" + hasMemberPredicate + "> ;";
+        final Response container = createIndirectContainer(uri, body);
+        final String containerResource = getLocation(container);
+        confirmPresenceOrAbsenceOfTripleInResponse(doGet(containerResource), containerResource,
+                                                   LDP_HAS_MEMBER_RELATION_PREDICATE, hasMemberPredicate, true);
+    }
+
+    /**
+     * 3.1.3-G
+     */
+    @Test(groups = {"MUST"})
+    public void ldpIndirectContainerMustAllowIsMemberOfRelationPredicateToBeSetOnCreate() {
+        setupTest("3.1.3-G",
+                  "Implementations must allow the membership predicate  to be set on indirect containers" +
+                  "via ldp:isMemberOfRelation property " +
+                  "of the content RDF on container creation.",
+                  "https://fcrepo.github.io/fcrepo-specification/#ldpic",
+                  ps);
+        //throw skip exception if indirect containers not supported
+        skipIfIndirectContainersNotSupported();
+
+        final String isMemberPredicate = "http://example.org/ldp/isMemberOf";
+        final String body = "<> <" + LDP_IS_MEMBER_OF_RELATION_PREDICATE + "> <" + isMemberPredicate + "> ;";
+        final Response container = createIndirectContainer(uri, body);
+        final String containerResource = getLocation(container);
+        confirmPresenceOrAbsenceOfTripleInResponse(doGet(containerResource), containerResource,
+                                                   LDP_IS_MEMBER_OF_RELATION_PREDICATE, isMemberPredicate, true);
+    }
+
+    /**
+     * 3.1.3-H
+     */
+    @Test(groups = {"MUST"})
+    public void ldpIndirectContainerMustAllowMembershipPredicateToBeSetByDefault() {
+        setupTest("3.1.3-H",
+                  "Implementations must allow the indirect container's membership predicate  to be set by " +
+                  "default to an implementation defined value. ",
+                  "https://fcrepo.github.io/fcrepo-specification/#ldpic",
+                  ps);
+        //throw skip exception if indirect containers not supported
+        skipIfIndirectContainersNotSupported();
+
+        final String body = "";
+        final Response container = createIndirectContainer(uri, body);
+        final String containerResource = getLocation(container);
+        final String responseBody = doGet(getLocation(container)).getBody().asString();
+        try (CloseableDataset dataset = parseTriples(IOUtils.toInputStream(responseBody))) {
+            final DatasetGraph graph = dataset.asDatasetGraph();
+            if (!graph
```

Comment:
Minor comment, but can the formatting of both `!graph.contains()` statements be made consistent?